Gemfile

```diff
@@ -5,12 +5,11 @@ ruby '2.7.0'
 
 gem 'rails', '6.1.3.1'
 gem 'token_master', '~> 1.0'
-gem 'sentry-raven', '~> 2.0'
 gem 'rubocop-performance'
 gem 'rubocop'
 gem 'rails_util', github: 'launchpadlab/rails_util'
 gem 'rack-cors', '~> 1.0'
-gem 'paper_trail', '~> 10.0'
+gem 'paper_trail', '~> 11.0'
```

Comment:
Upgraded to remove console warning about v10 not being compatible with Rails 6.1

Reply:
https://github.com/LaunchPadLab/rails_template/issues/124 😁